commandline/src/test/java/com/thoughtworks/go/util/command/CommandLineScriptRunnerTest.java

```diff
@@ -15,16 +15,25 @@
  */
 package com.thoughtworks.go.util.command;
 
+import com.thoughtworks.go.util.SystemEnvironment;
+import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.condition.DisabledOnOs;
 import org.junit.jupiter.api.condition.EnabledOnOs;
 import org.junit.jupiter.api.condition.OS;
 
+import static com.thoughtworks.go.util.SystemEnvironment.CONSOLE_LOG_MAX_LINE_LENGTH;
 import static java.nio.charset.StandardCharsets.UTF_8;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.fail;
 
 class CommandLineScriptRunnerTest {
+
+    @AfterEach
+    public void tearDown() {
+        System.clearProperty(CONSOLE_LOG_MAX_LINE_LENGTH.propertyName());
+    }
+
```

Comment:
The more modern of the tests use `SystemStub`s to ensure properties are restored to what they were before without lots of manual cruft. Could consider using these.

e.g

https://github.com/gocd/gocd/blob/0f58107c851cf2df6ce7c6902eebde796dc1f742/domain/src/test/java/com/thoughtworks/go/config/materials/git/GitMaterialTest.java#L70-L89

Reply:
I tweaked this in a subsequent commit to do some other cleanups. Should get around to moving everything...